src/app/modules/shopping-cart/components/containers/shopping-cart-details/shopping-cart-details.component.ts

```diff
@@ -1,12 +1,57 @@
-import { Component } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Component, OnInit } from '@angular/core';
+import { UntilDestroy } from '@ngneat/until-destroy';
+import { Observable, of } from 'rxjs';
 import { productsMockList } from 'src/app/components/mocks/products.mock';
-import { Product } from 'src/app/modules/shared/types/product';
+import { ShoppingCartService } from 'src/app/modules/shared/services/shopping.cart.service';
+import { OrderProductWithId } from 'src/app/modules/shared/types/order.product';
+import { OrderProduct } from 'src/app/modules/shared/types/product.order';
+import { OrderByCustomerDto } from 'src/app/modules/shared/types/product.order.with.customer';
+import { environment } from 'src/environments/environment';
 
+@UntilDestroy({
+  checkProperties: true,
+})
 @Component({
   selector: 'app-shopping-cart-details',
   templateUrl: './shopping-cart-details.component.html',
   styleUrls: ['./shopping-cart-details.component.scss'],
 })
 export class ShoppingCartDetailsComponent {
-  productList: Product[] = productsMockList;
+  private shoppingCartService: ShoppingCartService;
```

Comment:
Similar comment left above. You don't need to declare an attribute for an injected service in the component, Just inject it and then you can use it

Reply:
Seems like this is not solved yet